pkg/acsengine/params_k8s.go

```diff
@@ -19,36 +19,36 @@ func assignKubernetesParameters(properties *api.Properties, parametersMap params
 
 	if orchestratorProfile.IsKubernetes() ||
 		orchestratorProfile.IsOpenShift() {
-		k8sVersion := orchestratorProfile.OrchestratorVersion
+		kubeConfig := api.KubeConfigs[orchestratorProfile.OrchestratorVersion]
```

Comment:
This is a good optimization that avoids unnecessary lookups. 👍 

While we're here, let's do this, if you don't mind :)

1) rename `KubeConfigs` to `K8sComponentsByVersionDict` (or something)
2) generate the `K8sComponentsByVersionDict` map only once, instead of calling something like `getKubeConfigs()` every time it is referenced

Does that make sense?